package.json

```diff
@@ -128,8 +128,8 @@
     "node": "^14.15.0 || >=16.13.0"
   },
   "dependencies": {
-    "@aws-sdk/client-ec2": "3.35.0",
-    "@aws-sdk/client-ecr": "3.38.0",
+    "@aws-sdk/client-ec2": "3.46.0",
+    "@aws-sdk/client-ecr": "3.46.0",
```

Comment:
Are we properly grouping the `@aws-sdk/client-ec2`, `@aws-sdk/client-ecr` and `@aws-sdk/client-s3` packages in our `renovatebot/.github` rules, or in our own repo's `renovate.json` file?

Is there a way to prevent these packages to get out of sync with each other?

Reply:
@HonkingGoose hey are grouped, but they need dashborad aproval. They got out of sync, because some are added later in a PR and the main branch was already on a newer version.

Reply:
> @HonkingGoose they are grouped, but they need dashborad aproval. They got out of sync, because some are added later in a PR and the main branch was already on a newer version.

Are you sure these packages _are_ grouped together???


Quote from our `renovatebot/github/default.json` [^quote] file:

```json
    {
      "description": "Require approval for aws-sdk as it updates too often",
      "matchDatasources": ["npm"],
      "matchPackageNames": ["aws-sdk"],
      "matchPackagePatterns": ["^@aws-sdk\\/"],
      "dependencyDashboardApproval": true
    },
```

I do not see anything here that actually groups the `aws-sdk` packages together?

[^quote]: https://github.com/renovatebot/.github/blob/03cb56ff31a80b78b224e6024508007f502d77b6/default.json#L80-L86

Reply:
yes:
https://github.com/renovatebot/renovate/blob/2e258d81bd4bc905607bfeeb1d191446bf017024/lib/config/presets/internal/monorepo.ts#L24

Reply:
That line only groups any `aws-sdk-js-v3` packages, I think.

Would it be a good idea to have a `patternGroups` entry that puts these packages into one group?

- `@aws-sdk/client-ec2`
- `@aws-sdk/client-ecr`
- `@aws-sdk/client-s3`

https://github.com/renovatebot/renovate/blob/2e258d81bd4bc905607bfeeb1d191446bf017024/lib/config/presets/internal/monorepo.ts#L194-L203

Reply:
@HonkingGoose They are grouped via `sourceUrl`, checkout https://www.npmjs.com/package/@aws-sdk/client-ec2. All `@aws-sdk/*` js pacvkages are published from https://github.com/aws/aws-sdk-js-v3

Lets move this to a discussion if you need further clarification.